tools/include/eosio/abigen.hpp

```diff
@@ -257,6 +257,31 @@ namespace eosio { namespace cdt {
          _abi.tables.insert(t);
       }
 
+      void add_structure( const clang::CXXRecordDecl* decl ) {
```

Comment:
I'm a bit confused about this part.

I thought the idea of the new attribute was to flag data types to be included in the 'structs' section for abi conversions to be performed on it.

I don't see why we need another section that is duplicating the information.

Reply:
I noticed that eosio::table attribute is made available in both 'structs' section, and also in a separate section, and I assumed that different tools get that information from different sections. I was mimicking that implementation. I can remove the separate section, if my assumption is incorrect, leaving eosio::structure attribute only in 'structs' section. Please advice.

Reply:
Removed ABI generation of structures section avoiding duplication.